mods/ts/rules/gdi-support.yaml

```diff
@@ -166,7 +166,7 @@ GACSAM:
 		Cost: 300
 	Tooltip:
 		Name: SAM. Upgrade
-		Description: GDI Anti-Air base defense. \nRequires power to operate.\n  Strong vs all Aircraft\n  Cannot target ground units
+		Description: GDI Anti-Air base defense.\nRequires power to operate.\n  Strong vs Aircraft\n  Cannot target ground units
```

Comment:
There is a difference between drawing power and requiring power (becoming disabled when the player is in a low power state).